package.json

```diff
@@ -17,7 +17,8 @@
     "watch": "npm run watch:server & npm run watch:client",
     "build:server": "webpack --config webpack.config.js --config-name server",
     "build:client": "webpack --config webpack.config.js --config-name client",
-    "storybook": "start-storybook"
+    "storybook": "start-storybook -s ./public",
```

Comment:
What does `-s ./public` do? Does it put output in the repo `/public` directory or in a public directory inside `.storybook`?

Reply:
I've changed this and now uploaded the fonts to s3.

This would work running storybook locally but didn't work on GitHub pages (loading fonts)